client/assets/components/document/document_mail/document_mail.html

```diff
@@ -1,6 +1,6 @@
 <div class="detail vertical grid-content">
   <!-- show head field linked with a url. -->
-  <h3 class="head-field" ng-if="doc.subject"><a href="#"
+  <h3 class="head-field" ng-if="doc.subject"><a href="http://asfmail.lucidworks.io/"
```

Comment:
(in general, the "parent_s" field is the webpage which linked to the current one.  For the "raw" mail messages, it's always linked only from the same content in HTML form, so this is correct)